org-gtd-capture.el

```diff
@@ -37,7 +37,6 @@ Wraps the function `org-capture' to ensure the inbox exists.
 For GOTO and KEYS, see `org-capture' documentation for the variables of the same name."
   (interactive)
   (with-org-gtd-context
-      (org-gtd--ensure-inbox)
```

Comment:
Wait.. Removing this line doesn't break anything? That's strange, it feels like it should...  Processing the inbox doesn't work well if we don't make sure it has the default keywords and such available.

Reply:
Never mind, we just can't guarantee the user will use the inbox anymore I suppose.